011-kafka-canary.md

```diff
@@ -0,0 +1,40 @@
+# Strmixi Canary
+
+Implement a Kafka canary tool which will act as an indicator of whether Kafka clusters are operating correctly. This would be achieved by creating a canary topic and continously producing and consuming events on the topics. 
+
+## Current situation
+
+Currently  Strimzi does not have any canary style feature. It does have readiness checks for Kafka cluster which can be replaced/augmented with the canary.
+
+## Motivation
+
+The canary would provide the following:
+  
+  * Improved readiness check for Kafka clusters
+  * SLI(s) for service admins
+  * Metrics:
+    * End-to-end latency
```

Comment:
It is not really end-to-end latency from the customer perspective. It can be measured, but its value is limited. Not sure I would call it end-to-end latency.